packages/rest/src/validation/request-body.validator.ts

```diff
@@ -87,10 +87,17 @@ function convertToJsonSchema(openapiSchema: SchemaObject) {
  * @param schema The JSON schema used to perform the validation.
  * @param globalSchemas Schema references.
  */
+export interface AnyObject {
```

Comment:
AFAICT, this interface is not used anywhere. Can you remove it please?

Also please check the tsdoc comment above - it belongs to `validateValueAgainstJsonSchema`, not AnyObject!